src/http/APIAccessBase.php

```diff
@@ -4,13 +4,13 @@
 
 abstract class APIAccessBase implements APIAccessInterface {
     protected $apiKeyStorage;
-    protected $apiLogPackageEndpoint;
+    protected $apiSettings;
```

Comment:
I would recommend to keep class variables private and add explicit getter or setter for given variables.

Reply:
Done